src/engines/misc/sends/screen.cpp

```diff
@@ -26,304 +30,117 @@ namespace otto::engines::sends {
+    constexpr float y_dry = 78 + 26;
+    constexpr float y_fx2 = 130 + 26;
+
+    // Input
+    constexpr float input_radius = 30;
+    constexpr float input_x = 45;
+    ctx.beginPath();
+    ctx.circle({input_x, y_dry}, input_radius);
     ctx.stroke();
-
-    // sends/sendsIcon/circle
-    ctx.beginPath();
-    ctx.moveTo(41.7, 120.0);
-    ctx.bezierCurveTo(41.7, 126.0, 36.8, 130.9, 30.8, 130.9);
-    ctx.bezierCurveTo(24.8, 130.9, 19.9, 126.0, 19.9, 120.0);
-    ctx.bezierCurveTo(19.9, 114.0, 24.8, 109.1, 30.8, 109.1);
-    ctx.bezierCurveTo(36.8, 109.1, 41.7, 114.0, 41.7, 120.0);
-    ctx.closePath();
-    ctx.strokeStyle(Colours::Gray70);
-    ctx.stroke();
-
-    // sends/FX1
-    ctx.restore();
-
-    // sends/FX1/FX1
-    ctx.save();
-    ctx.beginPath();
-    ctx.moveTo(115.1, 27.8);
-    ctx.bezierCurveTo(114.2, 27.2, 112.9, 27.4, 112.2, 28.3);
-    ctx.lineTo(109.3, 32.4);
-    ctx.lineTo(106.4, 28.3);
-    ctx.bezierCurveTo(106.4, 28.3, 106.4, 28.3, 106.3, 28.2);
-    ctx.bezierCurveTo(106.3, 28.2, 106.2, 28.1, 106.2, 28.0);
-    ctx.bezierCurveTo(106.1, 28.0, 106.1, 27.9, 106.0, 27.9);
-    ctx.bezierCurveTo(106.0, 27.9, 105.9, 27.8, 105.9, 27.8);
-    ctx.bezierCurveTo(105.8, 27.7, 105.7, 27.7, 105.7, 27.7);
-    ctx.bezierCurveTo(105.6, 27.7, 105.6, 27.6, 105.5, 27.6);
-    ctx.bezierCurveTo(105.4, 27.6, 105.4, 27.6, 105.3, 27.5);
-    ctx.bezierCurveTo(105.2, 27.5, 105.2, 27.5, 105.1, 27.5);
-    ctx.bezierCurveTo(105.0, 27.5, 104.9, 27.5, 104.9, 27.5);
-    ctx.bezierCurveTo(104.8, 27.5, 104.8, 27.4, 104.7, 27.4);
-    ctx.lineTo(104.7, 27.4);
-    ctx.bezierCurveTo(104.7, 27.4, 104.7, 27.4, 104.7, 27.4);
-    ctx.lineTo(92.8, 27.4);
-    ctx.bezierCurveTo(91.7, 27.4, 90.7, 28.4, 90.7, 29.5);
-    ctx.lineTo(90.7, 42.5);
-    ctx.bezierCurveTo(90.7, 43.6, 91.7, 44.6, 92.8, 44.6);
-    ctx.bezierCurveTo(94.0, 44.6, 94.9, 43.6, 94.9, 42.5);
-    ctx.lineTo(94.9, 39.1);
-    ctx.lineTo(98.9, 39.1);
-    ctx.bezierCurveTo(100.1, 39.1, 101.0, 38.2, 101.0, 37.0);
-    ctx.bezierCurveTo(101.0, 35.9, 100.1, 34.9, 98.9, 34.9);
-    ctx.lineTo(94.9, 34.9);
-    ctx.lineTo(94.9, 31.6);
-    ctx.lineTo(103.6, 31.6);
-    ctx.lineTo(106.8, 36.0);
-    ctx.lineTo(103.0, 41.3);
-    ctx.bezierCurveTo(102.3, 42.2, 102.6, 43.5, 103.5, 44.2);
-    ctx.bezierCurveTo(103.9, 44.5, 104.3, 44.6, 104.7, 44.6);
-    ctx.bezierCurveTo(105.4, 44.6, 106.0, 44.3, 106.4, 43.7);
-    ctx.lineTo(109.3, 39.6);
-    ctx.lineTo(112.2, 43.7);
-    ctx.bezierCurveTo(112.6, 44.3, 113.3, 44.6, 113.9, 44.6);
-    ctx.bezierCurveTo(114.3, 44.6, 114.8, 44.5, 115.1, 44.2);
-    ctx.bezierCurveTo(116.1, 43.5, 116.3, 42.2, 115.6, 41.3);
-    ctx.lineTo(111.9, 36.0);
-    ctx.lineTo(115.6, 30.8);
-    ctx.bezierCurveTo(116.3, 29.8, 116.1, 28.5, 115.1, 27.8);
-    ctx.closePath();
+    ctx.fill(Colours::White);
+    // Icon
+    icon_.set_size({input_radius * 1.5, input_radius * 1.5});
+    icon_.set_color(Colours::Gray50);
+    icon_.set_line_width(6.f);
```

Comment:
no need to set these variables each frame, you can just set them in the constructor